abis_mapping/templates/dwc_mvp/mapping.py

```diff
@@ -1772,6 +1792,81 @@ def add_occurrence_status_value(
         graph.add((uri, rdflib.RDFS.label, rdflib.Literal(f"occurrenceStatus = {row['occurrenceStatus']}")))
         graph.add((uri, rdflib.RDF.value, VOCAB_OCCURRENCE_STATUS[row["occurrenceStatus"]]))
 
+    def add_establishment_means_observation(
+        self,
+        uri: rdflib.URIRef,
+        row: frictionless.Row,
+        dataset: rdflib.URIRef,
+        sample_field: rdflib.URIRef,
+        establishment_means_status_value: rdflib.URIRef,
```

Comment:
this can just be `establishment_means_value` rather than `establishment_means_status_value`